app/assets/stylesheets/common/admin/badges.scss

```diff
@@ -79,6 +79,11 @@
     .icon-picker {
       width: 350px;
     }
+
+    &__toggle-badge {
+      position: absolute;
```

Comment:
Will this overlap on mobile? I'm not a huge fan of using absolute positioning here tbh. 

I know we have some internal mockups showing the toggle right-aligned, like in the screenshot, but I wouldn't mind trying it out left-aligned, like the other fields, just below the title.

Reply:
Yeah I agree, I actually like it left-aligned. When its left-aligned its closer in proximity to the badge list so it makes it even easier to quickly toggle the switch while going through the list.

Did you want it below the name input? I'm not sure if moving it inside the `<form>` element will be a problem (since its has an independent action triggered instantly so you don't need to scroll and save), but if not I can do it like #1. What do you think?

**#1**
![Screenshot 2023-02-09 at 10 40 40](https://user-images.githubusercontent.com/30090424/217907134-d398f28d-4576-4402-a9ae-08c6979f7a0b.png)

**#2**
![Screenshot 2023-02-09 at 10 40 58](https://user-images.githubusercontent.com/30090424/217907193-944d7fe1-a23b-43ba-a101-815b71c0e9ed.png)

Reply:
Either #1 or #2 is fine, I don't have a strong preference.